runtime/lib/ttnn/program.cpp

```diff
@@ -17,6 +17,7 @@
 #include "operations/eltwise/unary/unary.h"
 #include "operations/eltwise/unary/unary_composite.h"
 #include "operations/embedding/embedding.h"
+#include "operations/embedding/embedding_backward.h"
```

Comment:
Would you mind adding this include (and the `embedding.h` one) to the `tools/ttnn-standalone/ttnn-precompiled.hpp`?